### PR TITLE
Auto-close PRs on subtree-splits

### PR DESCRIPTION
**Pull request by kbond**
Same as https://github.com/symfony/symfony/pull/54653

### CODE REVIEW

File: .github/sync-packages.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    file_put_contents($package.'/.gitattributes', $c);
+
+
+    @mkdir($package.'/.github');
+    file_put_contents($package.'/.github/PULL_REQUEST_TEMPLATE.md', <<<EOTXT
+        Please do not submit any Pull Requests here. They will be closed.
+        ---
+        Please submit your PR here instead:
+        {$mainRepo}
+        This repository is what we call a "subtree split": a read-only subset of that main repository.
```
Comment (nicolas-grekas) [inline .github/sync-packages.php:27]: missing blank line before :)

File: .github/sync-packages.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                script: |
+                  if (context.repo.owner === "twigphp") {
+                    github.rest.issues.createComment({
+                      owner: "twigphp",
+                      repo: context.repo.repo,
+                      issue_number: context.issue.number,
+                      body: `
+                  Thanks for your Pull Request! We love contributions.
+                  However, you should instead open your PR on the main repository:
+                  {$mainRepo}
```
Comment (nicolas-grekas) [inline .github/sync-packages.php:55]:
```suggestion

                  However, you should instead open your PR on the main repository:
                  {$mainRepo}

```

File: .github/workflows/package-tests.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  verify:
+    name: Verify Packages
+    runs-on: Ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Fetch branch from where the PR started
+        run: git fetch --no-tags --prune --depth=1 origin +refs/heads/*:refs/remotes/origin/*
+
```
Comment (nicolas-grekas) [inline .github/workflows/package-tests.yml:21]:
```suggestion
```

File: .github/sync-packages.php [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+                  `
+                    });
+                    github.rest.pulls.update({
+                      owner: "twigphp",
+                      repo: context.repo.repo,
+                      pull_number: context.issue.number,
+                      state: "closed"
+                    });
+                  }
+        EOTXT
```
Comment (nicolas-grekas) [inline .github/sync-packages.php:69]:
```suggestion

        EOTXT
```

**Comment by nicolas-grekas** [comment]
Thank you @kbond.

**Comment by fabpot** [comment]
Reverted

**Pull request merged**